src/components/api/devfile-registry.factory.ts

```diff
@@ -62,24 +71,18 @@ export class DevfileRegistry {
     this.headers = { 'Authorization': undefined };
   }
 
-  fetchDevfiles(location: string): ng.IPromise<Array<IDevfileMetaData>> {
-    let promise = this.$http({
-      'method': 'GET',
-      'url': `${location}/devfiles/index.json`,
-      'headers': this.headers
-    });
-
-    return promise.then((result: any) => {
-      return result.data.map((devfileMetaData: IDevfileMetaData) => {
-        let globalMemoryLimitNumber = this.getMemoryLimit(devfileMetaData.globalMemoryLimit);
-        // TODO remove this after fixing https://github.com/eclipse/che/issues/11424
-        if (this.isKeycloackPresent) {
-          globalMemoryLimitNumber += this.jwtproxyMemoryLimitNumber;
```

Comment:
We still have this issue https://github.com/eclipse/che/issues/11424 as an open one. Do we have memory differences between single and multiuser after these changes(shown and real one)?

Reply:
this part has been moved here
https://github.com/eclipse/che-dashboard/pull/5/files#diff-589d1bdb52ef3ad2061641d2cdd9b8edR182

Reply:
cool